packages/xo-server-sdn-controller/src/index.js

```diff
@@ -572,7 +540,12 @@ class SDNController extends EventEmitter {
         }
 
         const pif = find(host.$PIFs, { network: network.$ref })
-        if (pif != null && pif.currently_attached && host.enabled) {
+        if (
+          pif != null &&
+          pif.currently_attached &&
+          host.enabled &&
+          host.$metrics.live
+        ) {
           newCenter = host
         }
       })
```

Comment:
Split this loop into two:

- one sync loop to find the `newCenter`
- one parallel async loop to reset the network if necessary.

Reply:
Will do :+1: